exporter/awsxrayexporter/internal/translator/http.go

```diff
@@ -33,62 +47,63 @@ func makeHTTP(span ptrace.Span) (map[string]pcommon.Value, *awsxray.HTTPData) {
 
 	span.Attributes().Range(func(key string, value pcommon.Value) bool {
 		switch key {
-		case conventions.AttributeHTTPMethod:
+		case conventions.AttributeHTTPMethod, AttributeHTTPRequestMethod:
 			info.Request.Method = awsxray.String(value.Str())
 			hasHTTP = true
-		case conventions.AttributeHTTPClientIP:
+		case conventions.AttributeHTTPClientIP, AttributeClientAddress:
+			//urlParts[conventions.AttributeHTTPClientIP] = value.Str()
 			info.Request.ClientIP = awsxray.String(value.Str())
 			info.Request.XForwardedFor = aws.Bool(true)
 			hasHTTP = true
-		case conventions.AttributeHTTPUserAgent:
+		case conventions.AttributeHTTPUserAgent, AttributeUserAgentOriginal:
 			info.Request.UserAgent = awsxray.String(value.Str())
 			hasHTTP = true
-		case conventions.AttributeHTTPStatusCode:
+		case conventions.AttributeHTTPStatusCode, AttributeHTTPResponseStatusCode:
 			info.Response.Status = aws.Int64(value.Int())
 			hasHTTP = true
-		case conventions.AttributeHTTPURL:
-			urlParts[key] = value.Str()
+		case conventions.AttributeHTTPURL, AttributeURLFull:
+			urlParts[conventions.AttributeHTTPURL] = value.Str()
 			hasHTTP = true
 			hasHTTPRequestURLAttributes = true
-		case conventions.AttributeHTTPScheme:
-			urlParts[key] = value.Str()
+		case conventions.AttributeHTTPScheme, AttributeURLSchema:
+			urlParts[conventions.AttributeHTTPScheme] = value.Str()
 			hasHTTP = true
-		case conventions.AttributeHTTPHost:
-			urlParts[key] = value.Str()
+		case conventions.AttributeHTTPHost, AttributeServerAddress:
```

Comment:
Good question. Because http semantic conventions have been changed many times, v1.20 to v1.23.0 are just the latest changed. The change history is:
`http.host`(deprecated in v1.13.0) ->  `net.host.name`(deprecated) -> `server.address`

The awsxrayexporter translation is stale for a long time, this PR includes more changes than v.1.23.0.

Reply:
Please provide evidence of `http.host` -> `net.host.name`. I could only find https://github.com/open-telemetry/opentelemetry-specification/pull/2469, which says `http.host` -> `http.request.header.host`

Reply:
Fix this issue by adding `AttributeServerAddress` to urlParts, for building http server span URL later.
```
case AttributeServerAddress:
			urlParts[key] = value.Str()
			hasHTTPRequestURLAttributes = true
```